src/lib/field.unit-test.ts

```diff
@@ -55,6 +55,22 @@ test(Random.field, Random.int(-5, 5), (x, k) => {
   deepEqual(Field(x + BigInt(k) * Field.ORDER), Field(x));
 });
 
+// AND with some common and odd lengths
+[2, 4, 8, 16, 32, 64, 3, 5, 10, 15, Fp.sizeInBits].forEach((length) => {
+  test(Random.field, Random.field, (x_, y_, assert) => {
+    let x = x_ % BigInt(length);
+    let y = y_ % BigInt(length);
```

Comment:
take inputs modulo 2^length, see comment on the XOR PR: https://github.com/o1-labs/snarkyjs/pull/989/files#r1241966777